test/example/pr/test_task.wdl

```diff
@@ -0,0 +1,28 @@
+# Task Description
+# ================
+#
+# This task simply prints an input string "printed_string" to file and exposes that
+# file as "output_file".
+
+task Print {
+
+    String printed_string
+
+    command <<<
+
+        # print the string to file
```

Comment:
```suggestion
        # prints the string to file
```
Make a pointless change to adjust the hash, see if the circle tests re-run (they've hashed the previous, internal PR, which is pretty cool!)